### PR TITLE
app/st_things : Change folder structure for adding sample app

### PR DESCRIPTION
**Pull request by JeongJunSik**
- Change the structure so that you can build a sample app for future samples

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [6fa4bd0fd83df997b44c04c87ab2e3095ee90e5f] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [6fa4bd0fd83df997b44c04c87ab2e3095ee90e5f] - Code Rule Check OK.

File: apps/examples/st_things/Kconfig [hunk 1]
```diff
@@ -28,8 +39,10 @@ config RESET_BUTTON_PIN_NUMBER
...
 		If you use another board, you can set the pin number for the board.
 
+endchoice 
+
 endif
 
 config USER_ENTRYPOINT
 	string
-	default "st_things_sample_main" if EXAMPLES_ST_THINGS
+	default "st_things_sample_main" if EXAMPLES_ST_THINGS
```
Comment (davidfather) [inline apps/examples/st_things/Kconfig]: no newline at the end of file.

Comment (JeongJunSik) [reply apps/examples/st_things/Kconfig]: Thanks.

**Comment by seinfra** [comment]
Target : [ef53898004e1f2e35c2ff9673943dbfdae415bdc] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [ef53898004e1f2e35c2ff9673943dbfdae415bdc] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [5858cd3d6e7ed56ebfa5e2475f21eaa98d8547b0] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [5858cd3d6e7ed56ebfa5e2475f21eaa98d8547b0] - Code Rule Check OK.

**Pull request merged**